service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -276,4 +285,63 @@ private void buildCoreItem(String instanceId, String barcode, NodeList nodes, Ma
         catalogItems.put(barcode, itemData);
     }
 
+    @Override
+    public FeesFines getFeesFines(String uin) throws ParseException {
+        String path = "patron/account/";
+        String additional = "&includeLoans=false&includeCharges=true&includeHolds=false";
+        String url = String.format("%s%s%s?apikey={apikey}%s", getAPIBase(), path, uin, additional);
```

Comment:
This will not build the correct URL. `getAPIBase()` will return `protocol://host/app` with app being `oai` from getHoldings implementation. Will need some redesign of `getApiBase()` and methods using.

Reply:
I think `oai` path part is specific to that OAI endpoint and should not be in the config file.
`getApiBase()` is probably fine.
The code was originally written with the expectation that this was only going to talk to the OAI module.

Reply:
I understand. However, we now have even more knowledge mapping when using these configurations. The original schema of that map was for Voyager and should be adjusted for practical use with both. That said, the implementation of `getApiBase` should probably not include `app` and that be handled in Voyager specific scenarios.